### PR TITLE
Lockdown

### PR DESCRIPTION
**Pull request by Carson-Bay**

### CODE REVIEW

File: commands/lockdown.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        except:
+            replyEmbed = _embedMessage.create("Lockdown reply", "Invalid Role", "red")
+            await ctx.channel.send(embed = replyEmbed)
+            return
+        replyEmbed = _embedMessage.create("Lockdown Reply", "Channel Locked for {}".format(args[1]), "green")
+        await ctx.channel.send(embed = replyEmbed)
+        await ctx.channel.set_permissions(role, send_messages=False)
+
+    else:
+        replyEmbed = _embedMessage.create("Lockdown reply", "Something Fucky Happened", "red")
```
Comment (sahil-kale) [inline commands/lockdown.py:26]: Please change wording here

File: commands/lockdown.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from commands import _embedMessage, _checkrole, _util, _mongoFunctions
+from discord.utils import get
+from ._util import parse_message
+import discord
+
+async def lockdown(ctx: discord.message, client: discord.client):
+    if not (_checkrole.author_has_role(ctx, _mongoFunctions.get_admin_role(ctx.guild.id)) or _util.author_is_bot_owner(ctx)):
+        replyEmbed = _embedMessage.create("Locked Reply", "Invalid Permissions", "red")
```
Comment (sahil-kale) [inline commands/lockdown.py:8]: String should be Lockdown Reply

**Review by sahil-kale — CHANGES_REQUESTED** [review]

**Review by zaydtahir — CHANGES_REQUESTED** [review]
Can you add the two commands to $help? (Make sure they are in alphabetical order and under the admin section or Sahil will get angry)

**Review by sahil-kale — APPROVED** [review]
Approved

File: commands/help.py [hunk 3]
```diff
@@ -25,12 +25,14 @@ async def help_command(ctx, client):
     _embedMessage.add_field(helpMessage, commandPrefix2 + "adminverify @Mention",
                             "Manually verifies a user. Note that this does NOT add in a role and simply adds them to the database\nEx: " + commandPrefix2 + "adminverify " + client.user.mention,
                             False)
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "lockdown", "Sets send message permissions to false for specified role\nEx: " + commandPrefix2 + "lockdown " + "Tron", False)
```
Comment (zaydtahir) [inline commands/help.py:28]: The parameters of the command should be in the first string, like it is for all the other commands

File: commands/help.py [hunk 4]
```diff
@@ -25,12 +25,14 @@ async def help_command(ctx, client):
...
     _embedMessage.add_field(helpMessage, commandPrefix2 + "removeduedate", "Remove's a due date\nEx: " + commandPrefix2 + "removeduedate", False)
     _embedMessage.add_field(helpMessage, commandPrefix2 + "removequote \"quote with spaces\" Name",
                             "Removes a quote from the individual of your choice\nEx: " + commandPrefix2 + "addQuote \"Life is Good\", Bedi", False)
     _embedMessage.add_field(helpMessage, commandPrefix2 + "say title content channel",
                             "Sends a message inside an embed to the specified channel\nEx: " + commandPrefix2 + "say Hello world " + ctx.channel.mention, False)
     _embedMessage.add_field(helpMessage, commandPrefix2 + "setbedibotchannel",
                             "Sets the channel which will be used for announcements\nWARNING: This clears the channel's history. Use with caution.", False)
+    _embedMessage.add_field(helpMessage, commandPrefix2 + "unlock", "Sets send message permissions to True for specified role\nEx: " + commandPrefix2 + "unlock " + "Tron", False)
```
Comment (zaydtahir) [inline commands/help.py:36]: Same issue as previous comment

**Review by zaydtahir — CHANGES_REQUESTED** [review]
Just fix those two things and it should be good

**Review by sahil-kale — CHANGES_REQUESTED** [review]
Please resolve merge conflicts on main before merging

**Review by sahil-kale — APPROVED** [review]
Looks good to me with Zayd's approval

**Review by zaydtahir — APPROVED** [review]
Looks good

**Pull request merged**